src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -26,7 +26,10 @@ class DoctrineExtensionTest extends TestCase
      */
     private $extension;
 
-    protected function setUp()
+    /**
+     * @before
+     */
+    protected function before()
     {
         parent::setUp();
```

Comment:
all similar calls should also be updated

Reply:
Is `before()` available at [`TestCase`](https://github.com/sebastianbergmann/phpunit/blob/c1b8534b3730f20f58600124129197bf1183dc92/src/Framework/TestCase.php#L53)?

Reply:
no. But there is no parent to call at all. The parent setup will be called automatically by PHPunit (as we don't override it anymore with our own).